packages/react-docsite-components/src/components/PageHeader/PageHeader.tsx

```diff
@@ -39,15 +58,47 @@ const getStyles: IStyleFunction<IPageHeaderStyleProps, IPageHeaderStyles> = prop
 const getClassNames = classNamesFunction<IPageHeaderStyleProps, IPageHeaderStyles>();
 
 const PageHeaderBase: React.FunctionComponent<IPageHeaderProps> = props => {
-  const { pageTitle = 'Page title', pageSubTitle, theme } = props;
-  const styles = getClassNames(getStyles, { theme });
+  const { className, pageTitle = 'Page title', pageSubTitle, theme, versionSwitcherDefinition } = props;
+  const styles = getClassNames(getStyles, { className, theme });
+
+  const versionOptions: IContextualMenuItem[] | undefined =
+    versionSwitcherDefinition && versionSwitcherDefinition.versions
+      ? versionSwitcherDefinition.versions.map(version => ({
+          key: version,
+          text: version,
+          checked: version === versionSwitcherDefinition.currentVersion,
+        }))
+      : undefined;
 
   return (
-    <header className={css(styles.root, props.className)}>
+    <header className={styles.root}>
       <h1 className={styles.title}>
         {pageTitle}
         {pageSubTitle && <span className={styles.subTitle}>{pageSubTitle}</span>}
       </h1>
+      {versionSwitcherDefinition && versionOptions && (
+        <ActionButton
+          allowDisabledFocus={true}
+          styles={{
+            root: { height: '1em', padding: '12px 0' },
```

Comment:
Not set on this but I think it would look better to have the button text be the same color as "Web"

Also the vertical alignment is off
![image](https://user-images.githubusercontent.com/5864305/99319136-e10fbe00-281d-11eb-8443-4ce65ffed610.png)

Reply:
This should be solved now.